### DISCUSSION TITLE
Can we customize the 'html' format output to not include Graph/Chart for each file containing violations?

### DISCUSSION THREAD

**Discussion by vc4u** [forcedotcom/sfdx-scanner#1360] [Q&A]
Running a command like this:
`sf scanner run dfa --format=html --outfile=CodeAnalyzerDFA.html --target="./" --projectdir="./" --category="Security"`

Can we customize the html report generating template such that it does not output 'charts' for each code file containing violations?

**Comment by johnbelosf** [comment]
@vc4u not at the moment, but I'd love to better understand what you have in mind

**Reply by vc4u** [reply]
I was hoping, if as a user we get access to the template file which we can thus add/remove the template to control the aspect keeping the tokens that help generate data. Like chart could be the tokens that we just remove from the template and then output command can provide the --template as input and it generates the output. Right now, this is manual task that we do, as part of our operations, we generate html reports and then we edit the reports by removing/applying css override to simply hide the chart rows :-).

**Comment by johnbelosf** [comment]
Sounds good @vc4u - if you want to go ahead and create a new issue as a feature request, we can consider it for future implementation

**Reply by vc4u** [reply]
Added following feature issue request #1361
